demo/server/src/app.ts

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+import * as express from 'express';
+import * as path from 'path';
+import * as logger from 'morgan';
+import * as cookieParser from 'cookie-parser';
+import * as bodyParser from 'body-parser';
+import index from './routes/index';
+import * as favicon from 'serve-favicon';
+
+const app: express.Express = express();
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')))
+
+app.use('/', index);
+
+//catch 404 and forward to error handler
+app.use((req: express.Request, res: express.Response, next) => {
```

Comment:
I think the compiler can infer these types for `req, res, next`, we don't even these annotations :smile: